packages/block-library/src/media-text/index.js

```diff
@@ -152,12 +154,13 @@ export const settings = {
 			mediaType,
 			mediaUrl,
 			mediaWidth,
+			mediaId,
 		} = attributes;
 		const mediaTypeRenders = {
-			image: () => <img src={ mediaUrl } alt={ mediaAlt } />,
+			image: () => <img src={ mediaUrl } alt={ mediaAlt } className={ ( mediaId && mediaType === 'image' ) ? `wp-image-${ mediaId }` : null } />,
```

Comment:
Do we need a deprecation just for the `className`?

Reply:
cc @aduth

Reply:
In this case given that the class being added is an internal one we need the deprecations.